DataDog/datadog-agent
# [orchestrator] [process] isolating orchestrator settings

### What does this PR do?
Isolating orchestrator settings out of the `process_config`

### Motivation

Isolate orchestrator related topics from process.

### Additional Notes

### Describe your test plan

Unit Tests + manual test.
Manual test:
- Run the Process-agent with the old settings (deprecated, but backward compatible)
    `process_config.orchestrator_additional_endpoints` and `process_config.orchestrator_dd_url` 

- Run the Process-agent with the new settings 
    `orchestrator_explorer.orchestrator_additional_endpoints` and `orchestrator_explorer.orchestrator_dd_url`.


verify in logs/backend whether data is send.
